ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -309,6 +472,61 @@ export default {
         return { infoText: output.avoidCleave() };
       },
     },
+    {
+      id: 'DelubrumSav Lord Fateful Words',
+      // 97E: Wanderer's Fate, Pushes outward on Fateful Word cast
+      // 97F: Sacrifice's Fate, Pulls to middle on Fateful Word cast
+      // Labyrinthine Fate is cast and 1 second later debuffs are applied
+      // First set of debuffs go out 7.7 seconds before Fateful Word is cast
+      // Remaining set of debuffs go out 24.3 seconds before Fateful Word is cast
+      netRegex: NetRegexes.gainsEffect({ effectId: '97[EF]' }),
+      condition: (data, matches) => {
+        return (matches.target === data.me);
+      },
+      delaySeconds: (data, matches) => {
+        if (data.secondFates)
+          return 21;
+        data.secondFates = 1;
+        return 4;
```

Comment:
I think this might be cleaner as a timeline trigger before Fateful Words rather than having multiple delays?

Reply:
Would we need to store the result of the debuff in this trigger using the netregex and then use the timelineTrigger to attempt to collect that debuff?

Reply:
I see timelineTrigger would also be better because this event happens in a loop and given the 21 second delay could lead to the callout happening long after the phase is completed.